Directory.Packages.props

```diff
@@ -68,6 +68,9 @@
     <PackageVersion Include="Microsoft.Extensions.Configuration" Version="9.0.9" />
     <PackageVersion Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="9.0.9" />
     <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="9.0.9" />
+    <PackageVersion Include="Microsoft.Identity.Abstractions" Version="9.5.0" />
+    <PackageVersion Include="Microsoft.Identity.Web" Version="3.14.0" />
```

Comment:
nit - Identity team suggested (in the AOT internal channel) today that we should use 4.0.0.

Reply:
Pushed. Please take a look. Marking as resolved.